components/statistics/ChartBarBox/index.tsx

```diff
@@ -216,17 +220,28 @@ function ChartBarBox(props: PropsType) {
             <div className="w-full px-20  md:pr-30 md:pl-0 h-[230px] md:h-[475px]">
               {!loading && chartData?.length && (
                 <ParentSize>
-                  {({ width, height }) => (
-                    <BarChart
-                      type={type}
-                      width={width}
-                      height={height}
-                      data={chartData}
-                      days={currentDays}
-                      buckets={buckets}
-                      colorBlockchainMap={colorBlockchainMap}
-                    />
-                  )}
+                  {({ width, height }) => {
+                    const isMobile = width <= 640;
+
+                    const bottomAxisData = isMobile
+                      ? mobileBottomAxisData
+                      : DesktopBottomAxisData;
+                    const bottomAxis = bottomAxisData[currentDays];
+                    return (
+                      <BarChart
+                        width={width}
+                        height={height}
+                        data={chartData}
+                        buckets={buckets}
+                        isMobile={isMobile}
+                        bottomAxis={bottomAxis}
+                        colorBucketMap={colorBlockchainMap}
+                        paddingInner={days === 7 ? 0.3 : 0.46}
+                        paddingOuter={days === 90 ? 1 : 0.3}
+                        isDollarValue={type === 'volume'}
```

Comment:
it is better to pass a function to generate label in tooltip based on the value to be more general and reusable:
```suggestion
                        getLabel={(value)=> `$ ${value}`}}
```

Reply:
You're right. It's better, fixed it